config/ab_tests.yml

```diff
@@ -43,3 +43,7 @@ test:
   2022_01_subjects_filters_test:
     default: 1
     subjects_filters: 0
+visual:
+  2022_01_homepage_quick_links_test:
+    without_quick_links: 1
+    with_quick_links: 1
```

Comment:
So in the visual tests we want an equal chance of either variant being assigned?

Reply:
I think the subjects filters test will need to be covered too!

Reply:
You'll probably want to fix the variant used in these tests by setting one variant to 1 and the other to 0

Reply:
wil explain what ive done here, its much better now